redis/asyncio/client.py

```diff
@@ -171,6 +171,7 @@ def __init__(
         single_connection_client: bool = False,
         health_check_interval: int = 0,
         client_name: Optional[str] = None,
+        set_lib: bool = True,
```

Comment:
Any specific reason you made this optional on connect? I could see instead, sending it as part of the connect statement - and if the redis version doesn't support it, grabbing this in a try. WDYT?

Reply:
Good idea! 
Now it's as part of the connect statement, also you can set it using the client_setinfo command.